apps/emqx_bridge/src/schema/emqx_bridge_v2_schema.erl

```diff
@@ -85,7 +86,21 @@ post_request() ->
 
 api_schema(Method) ->
     EE = ?MODULE:enterprise_api_schemas(Method),
-    hoconsc:union(bridge_api_union(EE)).
+    APISchemas = registered_api_schemas(Method),
+    hoconsc:union(bridge_api_union(EE ++ APISchemas)).
+
+registered_api_schemas(Method) ->
+    %% We *must* do this to ensure the module is really loaded, especially when we use
+    %% `call_hocon' from `nodetool' to generate initial configurations.
+    _ = emqx_bridge_v2:module_info(),
```

Comment:
Nit: since the checks for exported functions are done inside `emqx_action_info`, I think this is not necessary here:

```suggestion
```